.github/pull_request.md

```diff
@@ -0,0 +1,25 @@
+# Descripion📜
+Please include a summary of the change and which issue is fixed.
+
+Fixes #<issue_number_here>
+
+
+# Type of Change 📝
+please delete options that are not relevant. In order to tick the checkbox just put x inside them for example [x] like this
+
+- [ ] Add a new project
+- [ ] Modified or enchanced the existing projects
+- [ ] Improve the readme of any project
+
+# Domain of Contribution 📊
+please delete options that are not relevant. In order to tick the check box just put x inside them for example [x] like this
```

Comment:
Remove the 16 , 17 and 18 line and add  this 

[ ] Project using only HTML  ,CSS , JAVASCRIPT , WEB3.js
[ ] Project using only HTML , CSS , JAVASCRIPT , ETHER.js
[ ] Project using only HTML  , CSS , PYTHON , WEB3.py
[ ] Project using only GANACHE , TRUFFLE , HARDHAT , REMIX 
[ ] if Project use another stack and Tool feel free to mention